packages/template-retail-react-app/app/components/display-price/index.test.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {screen} from '@testing-library/react'
+import DisplayPrice from '@salesforce/retail-react-app/app/components/display-price/index'
+import {renderWithProviders} from '@salesforce/retail-react-app/app/utils/test-utils'
+
+describe('DisplayPrice', function () {
+    test('should render without error', () => {
+        renderWithProviders(<DisplayPrice currency="GBP" basePrice={100} discountPrice={90} />)
+        expect(screen.getByText(/£90\.00/i)).toBeInTheDocument()
+        expect(screen.getByText(/£100\.00/i)).toBeInTheDocument()
+    })
+
+    test('should render according html tag for prices', () => {
+        const {container} = renderWithProviders(
+            <DisplayPrice currency="GBP" basePrice={100} discountPrice={90} />
+        )
+        const discountPriceTag = container.querySelectorAll('s')
+        const basePriceTag = container.querySelectorAll('b')
```

Comment:
I think these be switched, the `basePriceTag` is the one that has a strikethrough, and the `discountPriceTag` is the bolded one right? Do we also want to add a check that the price in these tags line up with the base/discounted price we pass in?

Reply:
Good catch, more assertion has been added